GrowthViz-adults.py

```diff
@@ -116,23 +116,38 @@
 
 # Next, the `charts.setup_individual_obs_df` function performs transformations on the `cleaned_obs` DataFrame. This will create an `age` column, which is a decimal column that represents the patient's age in years at the time of the observation. It changes the `clean_value` column into a [pandas categorical type](https://pandas.pydata.org/pandas-docs/stable/user_guide/categorical.html). It also create an `include` column which contains a boolean value indicating whether growthcleanr states to include (true) or exclude (false) the observation. The resulting DataFrame is assigned to `obs`.
 
-# In[36]:
+# In[9]:
 
 
 obs_full = charts.setup_individual_obs_df(cleaned_obs)
 
 
 # In the following cell, the `charts.keep_age_range` function visually displays the range of ages in the dataset, with those to be excluded identified by the red bars. It then removes patients outside the intended target population of this notebook (adults 20 to 65).
 
-# In[43]:
+# In[10]:
 
 
 obs = charts.keep_age_range(obs_full)
```

Comment:
Have you looked at this with the axes rotated, with age bins on the x-axis and counts on the y-axis? I usually find it  intuitive to see anything time-related on the x-axis.